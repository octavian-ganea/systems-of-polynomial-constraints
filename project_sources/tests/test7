% Test 7 :
x1^2 - 4 * x1 + 4 + x2^2 - 4 * x2 + 4 + x3^2 - 4 * x3 + 4 - 1  = 0
