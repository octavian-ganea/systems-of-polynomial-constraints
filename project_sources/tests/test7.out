Parsing file in.txt
Parsing succeded. Results:
List of all variables:
Vector(x1, x2, x3)
List of all polynomials:
Vector(((11.0  + -4.0 x3 + 1.0 x3^2) + 1.0 x2^2 + -4.0 x2) + -4.0 x1 + 1.0 x1^2)

List of desired signs : Vector(=)
Analysed polynomials: 
((11.0  + -4.0 x3 + 1.0 x3^2) + 1.0 x2^2 + -4.0 x2) + -4.0 x1 + 1.0 x1^2
vars: Vector(x1, x2, x3)

Univariate projection (on real axis) contains these polynomials: 
11.0  + -4.0 x3 + 1.0 x3^2
28.0  + -16.0 x3 + 4.0 x3^2
768.0  + -1024.0 x3 + 256.0 x3^2
1808.0  + -896.0 x3 + 480.0 x3^2 + -128.0 x3^3 + 16.0 x3^4
228.0  + -352.0 x3 + 152.0 x3^2 + -32.0 x3^3 + 4.0 x3^4
121.0  + -88.0 x3 + 38.0 x3^2 + -8.0 x3^3 + 1.0 x3^4
784.0  + -896.0 x3 + 480.0 x3^2 + -128.0 x3^3 + 16.0 x3^4

There are 25 cells :
Map(x1 -> 0.0, x2 -> 0.0, x3 -> 0.0)
Map(x1 -> 0.0, x2 -> 1.0, x3 -> 1.0)
Map(x1 -> 1.0, x2 -> 2.0, x3 -> 1.0)
Map(x1 -> 2.0, x2 -> 2.0, x3 -> 1.0)
Map(x1 -> 3.0, x2 -> 2.0, x3 -> 1.0)
Map(x1 -> 0.0, x2 -> 3.0, x3 -> 1.0)
Map(x1 -> 0.0, x2 -> 0.0, x3 -> 2.0)
Map(x1 -> 1.0, x2 -> 1.0, x3 -> 2.0)
Map(x1 -> 2.0, x2 -> 1.0, x3 -> 2.0)
Map(x1 -> 3.0, x2 -> 1.0, x3 -> 2.0)
Map(x1 -> 0.0, x2 -> 2.0, x3 -> 2.0)
Map(x1 -> 1.0, x2 -> 2.0, x3 -> 2.0)
Map(x1 -> 2.0, x2 -> 2.0, x3 -> 2.0)
Map(x1 -> 3.0, x2 -> 2.0, x3 -> 2.0)
Map(x1 -> 4.0, x2 -> 2.0, x3 -> 2.0)
Map(x1 -> 1.0, x2 -> 3.0, x3 -> 2.0)
Map(x1 -> 2.0, x2 -> 3.0, x3 -> 2.0)
Map(x1 -> 3.0, x2 -> 3.0, x3 -> 2.0)
Map(x1 -> 0.0, x2 -> 4.0, x3 -> 2.0)
Map(x1 -> 0.0, x2 -> 1.0, x3 -> 3.0)
Map(x1 -> 1.0, x2 -> 2.0, x3 -> 3.0)
Map(x1 -> 2.0, x2 -> 2.0, x3 -> 3.0)
Map(x1 -> 3.0, x2 -> 2.0, x3 -> 3.0)
Map(x1 -> 0.0, x2 -> 3.0, x3 -> 3.0)
Map(x1 -> 0.0, x2 -> 0.0, x3 -> 4.0)
====== FINAL RESULT FOR in.txt ================================
Map(x1 -> 2.0, x2 -> 2.0, x3 -> 1.0)
====================================================

